packages/nouns-contracts/contracts/governance/fork/newdao/governance/NounsDAOLogicV1Fork.sol

```diff
@@ -207,17 +207,26 @@ contract NounsDAOLogicV1Fork is UUPSUpgradeable, ReentrancyGuardUpgradeable, Nou
             nouns.transferFrom(msg.sender, address(timelock), tokenIds[i]);
         }
 
+        address[] memory erc20TokensToIncludeInQuit_ = erc20TokensToIncludeInQuit;
+        uint256[] memory balancesToSend = new uint256[](erc20TokensToIncludeInQuit_.length);
+
+        // Capture balances to send before actually sending them, to avoid reentrancy risk
+        uint256 ethToSend = (address(timelock).balance * tokenIds.length) / totalSupply;
         for (uint256 i = 0; i < erc20TokensToIncludeInQuit.length; i++) {
```

Comment:
Can be replaced with `erc20TokensToIncludeInQuit_` to save SLOADs in L215 and 216